DeliveryServiceEF/DeliveryServiceEF.Web/Controllers/ProductViewController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using DeliveryServiceEF.Domain.Interfaces;
+using DeliveryServiceEF.Domain.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DeliveryServiceEF.Web.Controllers
+{
+    [Route("mvc/product")]
+    public class ProductViewController : Controller
+    {
+        private readonly IProductService _productService;
+
+        public ProductViewController(IProductService productService)
+        {
+            _productService = productService;
+        }
+
+        [HttpGet]
+        public IActionResult Index()
+        {
+            return View("Index", _productService.GetProducts());
+        }
+
+        [HttpGet("details")]
```

Comment:
[HttpGet("details")]
->
[HttpGet]

cause method already has name "Details"

Reply:
do the same with others please

Reply:
remove name from attribute if it is the same with method name

Reply:
Decided to keep as is